### ISSUE TITLE
Custom primary color with meteor

### ISSUE THREAD

**Issue by hmeerlo** [mdbootstrap/mdb-ui-kit#332]
Hi,

the README states that to use a custom primary color we have to do something like this:

@import "../bower_components/bootstrap-material-design/less/material-wfont.less";

But this doesn't work in the meteor case, what is the meteor alternative way of doing this?

/Herman

**Comment by FezVrasta** [comment]
Not sure how to change it with Meteor..

ping @dandv

**Comment by kaoskeya** [comment]
The meteor package includes css files and not less. This might not be possible, yet.

My, kinda messy, workaround is as given below:

project_folder
 | -- bower_components <- Created when "bower install bootstrap-material-design --save" is run in the project_folder
 | -- app <- the meteor app

And in my app/client folder, a less file with:

```
@import "../bower_components/bootstrap-material-design/less/material-wfont.less";
@import "../bower_components/bootstrap-material-design/less/ripples.less";
@primary: #673BB7;
```

In the client folder, did a soft link to the bower_components/bootstrap-material-design/dist/js folder

Template where I wanted the material JS magic to happen, added the following to the rendered function

```
$.material.init()
```

Kinda works for the moment. When there is an update on this repo, a 'bower update bootstrap-material-design' can be run. Dependencies are the https://github.com/Nemo64/meteor-bootstrap and less of course.

Ideally could be done as a meteor package, I'll try and get one done maybe.

**Comment by hmeerlo** [comment]
Ok thanx, that was exactly the workaround I made too :-)

**Comment by sasikanth513** [comment]
@dandv +1 for this in meteor

**Comment by FezVrasta** [comment]
merged with this other issue: #453
